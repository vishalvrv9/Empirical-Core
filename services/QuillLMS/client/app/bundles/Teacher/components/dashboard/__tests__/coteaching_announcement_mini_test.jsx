import React from 'react';
import { shallow } from 'enzyme';

import CoteachingAnnouncementMini from '../coteaching_announcement_mini';

describe('CoteachingAnnouncementMini component', () => {

  it('should render', () => {
    expect(shallow(<CoteachingAnnouncementMini />)).toMatchSnapshot();
  });

});
