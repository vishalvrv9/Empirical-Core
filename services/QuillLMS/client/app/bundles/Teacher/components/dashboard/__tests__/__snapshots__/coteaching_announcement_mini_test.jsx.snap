// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoteachingAnnouncementMini component should render 1`] = `
<div
  className="mini_container results-overview-mini-container col-md-4 col-sm-5 text-center"
>
  <div
    className="mini_content"
  >
    <div
      className="gray-underline"
      style={
        Object {
          "padding": "8px",
        }
      }
    >
      <img
        className="mini-star-icon"
        src="https://assets.quill.org/images/icons/icon-star.svg"
        style={
          Object {
            "height": "27px",
            "position": "absolute",
            "right": "6px",
            "top": "-11px",
            "transform": "rotate(-22deg)",
            "width": "27px",
          }
        }
      />
      <img
        className="mini-star-icon"
        src="https://assets.quill.org/images/icons/icon-star.svg"
        style={
          Object {
            "height": "14px",
            "position": "absolute",
            "right": "25px",
            "top": "-10px",
            "transform": "rotate(-4deg)",
            "width": "14px",
          }
        }
      />
      <h3>
        Co-Teaching is Now Available
      </h3>
    </div>
    <div>
      <img
        src="https://assets.quill.org/images/illustrations/high-five-illustrationx2.png"
        style={
          Object {
            "marginBottom": 11,
            "marginTop": 13,
            "width": 57,
          }
        }
      />
    </div>
    <p
      style={
        Object {
          "fontFamily": "lucida-grande, adelle-sans, helvetica",
          "lineHeight": "1.57",
          "padding": "0px 15px",
        }
      }
    >
      You can now invite co-teachers to your classes. Co-teachers can manage classes, assign activities and view reports.
    </p>
    <a
      href="/teachers/classrooms#invite-coteachers"
      style={
        Object {
          "display": "block",
          "marginBottom": "12px",
        }
      }
    >
      <button
        className="button button-white beta"
        style={
          Object {
            "margin": "15px auto 0px",
            "maxWidth": "233px",
            "width": "100%",
          }
        }
      >
        Invite Co-Teachers
      </button>
    </a>
    <a
      href="http://support.quill.org/getting-started-for-teachers/manage-classes/how-do-i-share-a-class-with-my-co-teacher"
      style={
        Object {
          "color": "#027360",
        }
      }
      target="_blank"
    >
      Learn more about co-teaching &gt;
    </a>
  </div>
</div>
`;
